core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -11970,5 +11970,28 @@ RelNode createLogicalValueRel(RexNode col1, RexNode col2) {
 
     assertThat(toSql(root, DatabaseProduct.ORACLE.getDialect()), isLinux(expectedOracle));
   }
+  
+  @Test public void testCountSetWithLiteralParameter() {
+    RelBuilder builder = relBuilder().scan("EMP");
+    final RexNode rexNode = builder.call(SqlLibraryOperators.BIT_COUNT,
+        builder.literal(7));
+    RelNode root = builder
+        .project(rexNode)
```

Comment:
Please also add an alias in the projection, so that translatedBqSql gets simple alias instead of `$f0`.
ex: .project(builder.alias(rexNode, "aa"))

Reply:
Done